pytorch/inference/docker/1.9/py3/cu111/Dockerfile.gpu

```diff
@@ -0,0 +1,204 @@
+FROM nvidia/cuda:11.1.1-base-ubuntu20.04
```

Comment:
Any particular reason to use the `base` flavor instead of `devel`?

Reply:
There are 2 reasons:
1. devel image comes with a lot of packages already installed and we don't use all the packages. 
2. devel image consumes more space as compared to base image.